examples/conf-quickstart/druid/_common/common.runtime.properties

```diff
@@ -41,26 +41,26 @@ druid.startup.logging.logProperties=true
 # Zookeeper
 #
 
-druid.zk.service.host=localhost
+druid.zk.service.host=localhost:2181
 druid.zk.paths.base=/druid
 
 #
 # Metadata storage
 #
 
 # For Derby server on your Druid Coordinator (only viable in a cluster with a single Coordinator, no fail-over):
-druid.metadata.storage.type=derby
-druid.metadata.storage.connector.connectURI=jdbc:derby://localhost:1527/var/druid/metadata.db;create=true
-druid.metadata.storage.connector.host=localhost
-druid.metadata.storage.connector.port=1527
+#druid.metadata.storage.type=derby
+#druid.metadata.storage.connector.connectURI=jdbc:derby://localhost:1527/var/druid/metadata.db;create=true
+#druid.metadata.storage.connector.host=localhost
+#druid.metadata.storage.connector.port=1527
 
 # For MySQL:
-#druid.metadata.storage.type=mysql
-#druid.metadata.storage.connector.connectURI=jdbc:mysql://db.example.com:3306/druid
-#druid.metadata.storage.connector.user=...
-#druid.metadata.storage.connector.password=...
+druid.metadata.storage.type=mysql
+druid.metadata.storage.connector.connectURI=jdbc:mysql://localhost:3306/druid
+druid.metadata.storage.connector.user=root
+druid.metadata.storage.connector.password=1234
```

Comment:
did you put your test files in the PR? If you want't to change the defaults in this config, you can raise another PR to do it.